packages/transaction-controller/src/utils.ts

```diff
@@ -105,6 +105,16 @@ export function validateTxParams(txParams: TransactionParams) {
       );
     }
   }
+
+  if (
```

Comment:
As mentioned in the other PR, could this also go into the new `validation.ts` as an internal function?

Reply:
Sure this make sense to me. I will fix this once Vini's PR land in.